tests/virt_autotest/virt_utils.pm

```diff
@@ -302,4 +302,79 @@ sub upload_supportconfig_log {
     save_screenshot;
 }
 
+# Download guest image and xml from a NFS location to local
+# the image and xml is coming from a guest installation testsuite
+# need set SKIP_GUEST_INSTALL=1 in the test suite settings
+# only available on x86_64
+sub download_guest_assets {
+
+    # guest_pattern is a string, like sles-11-sp4-64, may or may not with pv or fv given.
+    my ($guest_pattern, $vm_xml_dir) = @_;
+
+    # list the guests matched the pattern
+    my $qa_guest_config_file = "/usr/share/qa/virtautolib/data/vm_guest_config_in_vh_update";
+    my $guest_list           = script_output "source /usr/share/qa/virtautolib/lib/virtlib; get_vms_from_config_file $qa_guest_config_file $guest_pattern";
+    my $install_guest_list   = "";
+    if (get_var('SYSTEM_ROLE', '') eq "kvm") {
+        foreach my $vm (split '\n', $guest_list) {
+            if ($vm =~ /([\w\-]*-fv-[\w\-]*)/) {
+                $install_guest_list .= "$vm ";
+            }
+        }
+    }
+    else {
+        $install_guest_list = $guest_list;
+    }
+    save_screenshot;
+
+    # mount the remote NFS location of guest assets
+    # OPENQA_URL="localhost" in local openQA instead of the IP, so the line below need to be turned on and set to the webUI IP when you are using local openQA
+    # Tips: Using local openQA, you need "rcnfsserver start & vi /etc/exports; exportfs -r")
+    # set_var('OPENQA_URL', "your_ip");
+    my $openqa_server = get_required_var('OPENQA_URL');
+    $openqa_server =~ s/^http:\/\///;
+    my $remote_export_dir = "/var/lib/openqa/factory/other/";
+    my $mount_point       = "/tmp/remote_guest";
+
+    assert_script_run "if [ ! -d $mount_point ];then mkdir -p $mount_point;fi";
+    save_screenshot;
+    # tip: nfs4 is not supported on sles12sp4
+    assert_script_run("mount -t nfs $openqa_server:$remote_export_dir $mount_point", 120);
```

Comment:
You can refer to prj3 guest migration code to see which general mount command is supported on all products.

Reply:
I found that command(server is my laptop, client is a vhXXX with sles12sp4) did not work, so I changed to this one. 
mount -t nfs4 10.67.18.247:/var/lib/openqa/factory/other/ /tmp/remote_guest ↙ //
mount.nfs4: mounting 10.67.18.247:/var/lib/openqa/factory/other/ failed, reason given by server: No such file or directory
I have no sles12sp4 machine in hand, so I am unable to test it again. So just keep the current command if it is not incorrect.

Reply:
"mount -t nfs -o vers=4,nfsvers=4 ..." is used in prj3. what's the advantage with '-o vers=4,nfsvers=4'? some mount commands without it exist in our test script as well.
vm-migrate.sh:    $sshNoPass $migrateeUser@$migrateeIP "mount -t nfs $localIp:$domainDiskDir $domainDiskDir" 2> /dev/null